odin/adapters/adapter.py

```diff
@@ -80,6 +89,56 @@ def cleanup(self):
         pass
 
 
+class ApiAdapterRequest(object):
+    """API Adapter Request object.
+
+    Designed to emulate the HTTP Request Object used in the Get and Put requests
```

Comment:
Suggest this should read 

'Emulates the HTTPServerRequest' class passed as an argument to adapter HTTP verb methods (GET, PUT, ...)'